ci/docker-release-pipeline.yml

```diff
@@ -0,0 +1,54 @@
+trigger:
+  branches:
+    include:
+      - 'patch/*'
+
+pool:
+  vmImage: 'ubuntu-20.04'
+variables:
+  tag: '$(Build.SourceBranchName)'
+
+jobs:
+  - job: RunOn
+    displayName: "Build and push docker"
+
+    pool:
+      name: "Azure Pipelines"
+      vmImage: "ubuntu-latest"
+
+    timeoutInMinutes: 60
+
+    steps:
+      - bash: |
+          export hub_tag_name=$(git describe --tags)-$(echo "$(tag)" | sed -e 's/^patch\///' | sed -e 's/\//-/g')  
+          echo hub_tag_name=$hub_tag_name
+          echo "##vso[task.setvariable variable=hub_tag_name]$hub_tag_name"
+        name: check_changes
+        displayName: "Check changed files"
+
+      - task: Docker@2
+        inputs:
+          command: 'login'
+          containerRegistry: quest_docker_hub
+
+      - task: CmdLine@2
```

Comment:
When would this task run? Periodically (once per night) or on push to a `patch/*` branch? I can only see the push being the trigger. Am I missing something?

Reply:
"Nightly" in the task and the tag is misleading then. Maybe we should call it "snapshot" or "canary"?

Reply:
Why you don't like nightly? My intention is to schedule it nightly when PR merged to master

Reply:
If we run it each night, I'm more than fine with "nightly". I thought that this would be somehow configured in the pipeline definition.